coderd/users.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/coder/coder/coderd/userpassword"
 	"github.com/coder/coder/coderd/util/slice"
 	"github.com/coder/coder/codersdk"
-	"github.com/coder/coder/examples"
+	examples "github.com/coder/coder/templates"
```

Comment:
@ericpaulsen Shouldn't this specifically be the `quickstarts` folder since this is what we want to show in `coder templates init`?

Reply:
@bpmct yeah, quickstarts makes the most sense here for `coder templates init`.

Reply:
I think we need to change this to only reference the quickstarts since it's used for the autoimport behavior. I might be missing something though?

Reply:
I guess, I'm asking if we should change it to this 

```
quickstart "github.com/coder/coder/templates/quickstart"
```